File: php7/src/ChainOfResponsibility/Tests/Bank/LoanSystemBuilderTest.php
<?php

declare(strict_types = 1);

namespace PetrePatrasc\ChainOfResponsibility\Tests\Bank;


use PetrePatrasc\ChainOfResponsibility\Bank\LoanSystemBuilder;

class LoanSystemBuilderTest extends \PHPUnit_Framework_TestCase
{
    public function testBuild()
    {
        $loanSystem = LoanSystemBuilder::build();

        $this->assertNotNull($loanSystem);
        $this->assertInstanceOf('\PetrePatrasc\ChainOfResponsibility\Bank\LoanSystem', $loanSystem);
    }
}

File: java/src/ro/petrepatrasc/chainofresponsability/tests/bank/LoanSystemBuilderTest.java
package ro.petrepatrasc.chainofresponsability.tests.bank;

import junit.framework.TestCase;
import org.junit.Test;
import ro.petrepatrasc.chainofresponsability.bank.LoanSystem;
import ro.petrepatrasc.chainofresponsability.bank.LoanSystemBuilder;

public class LoanSystemBuilderTest extends TestCase {

    @Test
    public void testBuild() {
        LoanSystem loanSystem = LoanSystemBuilder.build();

        this.assertNotNull(loanSystem);
        this.assertTrue(loanSystem instanceof LoanSystem);
    }
}
